src/main/java/com/wavefront/sdk/direct/ingestion/WavefrontDirectIngestionClient.java

```diff
@@ -56,6 +58,7 @@
     private int maxQueueSize = 50000;
     private int batchSize = 10000;
     private int flushIntervalSeconds = 1;
+    private Integer messageSizeKilobytes = null;
```

Comment:
I'm assuming this is the variable that tells the SDK what sized packets to create to send. Can we name this to something like `flushSizeKbs`? It's a little confusing when we see two variables: `messageSizeBytes` and `messageSizeKilobytes`.

Is there a reason one is `bytes` and the other is `KBs`?

Reply:
Updated to use `bytes` everywhere.